src/test/java/restfulbooking/BookingJsonTest.java

```diff
@@ -0,0 +1,43 @@
+package restfulbooking;
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class BookingJsonTest
+{
+	DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
```

Comment:
This is just a sanity test for JSON conversion.  They will be removed out into a separate project when the API client is complete.